### DISCUSSION TITLE
Slint 1.2 Released with Enhanced Platform Abstraction

### DISCUSSION THREAD

**Discussion by tronical** [slint-ui/slint#3402] [Blog Comments]
Discussion for the Blog post [Slint 1.2 Released with Enhanced Platform Abstraction](https://slint.dev/blog/slint-1.2-released).

**Comment by Horbin-Magician** [comment]
Congratulations!
I would like to know whether you have the next updating roadmap or focuses?

**Reply by hunger** [reply]
We want to do some planning soon, we'll keep you posted :-)
